ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -52,6 +57,23 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
       this.can.can('connect target', target),
     );
 
-    return { targets: targets, totalItems };
+    let targetsWithSessions = [];
+    for (const target of targets) {
+      let sessions = await this.store.query('session', {
+        query: {
+          filters: {
+            target_id: { equals: target.id },
+            status: [
+              { equals: STATUS_SESSION_ACTIVE },
```

Comment:
Does equals only take a string? wondering if it can take in an array so we can combine these statuses in an array

Reply:
i was mainly following the format from this [example](https://github.com/hashicorp/boundary-ui/blob/llb/client-daemon-integration/addons/api/addon/utils/mqlQuery.js#L13). Based on the mqlQuery util, seems like each entry needs to be an obj with an operation and a value. so something like { equals: ['active', 'pending'] } wouldn't get parsed correctly.

Reply:
Yeah I already did it by having the key take in an array. Originally I had it as just `status: ['active', 'pending']` but I needed a bit more flexibility in cases where it's not a strict equality that we need. 

I could've had it default to equals if they're not objects but decided to make it explicit 🤔 Maybe an improvement to make if we see this being a hassle